OpenRA.Mods.Common/Traits/AttackMove.cs

```diff
@@ -153,7 +162,12 @@ protected virtual IEnumerable<Order> OrderInner(World world, CPos cell, MouseInp
 		public override string GetCursor(World world, CPos cell, int2 worldPixel, MouseInput mi)
 		{
 			if (world.Map.Contains(cell))
+			{
+				if (subjects.Any(a => !a.Actor.Info.TraitInfo<AttackMoveInfo>().MoveIntoShroud && !a.Actor.Owner.Shroud.IsExplored(cell)))
```

Comment:
I'm not sure whether it would be better to show this when *any* actor is blocked vs when *all* actors are blocked.  It would be good to get more opinions here.

Reply:
This can be simplified and unnecessary trait lookups can be removed:
```
if (world.Map.Contains(cell))
{
	var prefix = mi.Modifiers.HasModifier(Modifiers.Ctrl) ? "assaultmove" : "attackmove";

	var explored = subjects.First().Actor.Owner.Shroud.IsExplored(cell);
	var blocked =  !explored && subjects.Any(a => !a.Trait.Info.MoveIntoShroud);
	return blocked ? prefix + "-blocked" : prefix;
}
```

Plus then making the `info` field in AttackMove public.

Reply:
I am unsure about the _any_ either. Updated to avoid trait lookups.

Reply:
`.Any` gives this the same behaviour as the normal move cursor, so lets leave it how it is.